client/model.go

```diff
@@ -1225,3 +1225,16 @@ func (e *AuditLogEntry) DecisionLog() (*auditv1.DecisionLogEntry, error) {
 type PlanResourcesResponse struct {
 	*responsev1.PlanResourcesResponse
 }
+
+type (
+	ListPoliciesOption func(opts *ListPoliciesParams)
+	ListPoliciesParams struct {
+		IncludeDisabled bool
+	}
```

Comment:
I don't think you need this struct. Just define `ListPoliciesOption` as `func(*requestv1.ListPoliciesRequest)`.